### ISSUE TITLE
The solution does not show url's

### ISSUE THREAD

**Issue by jenokizm** [jasonfah/chrome-tab-titles#3]
Your sample code is incomplete. There is also a useful opportunity to get the url addresses of open tabs.

**Comment by jasonfah** [comment]
That's right, it's for tab titles only. You should be able to use a similar method though to get the address bar text.
